src/disp_s1/browse_image.py

```diff
@@ -18,18 +101,64 @@ def make_browse_image(
     ----------
     output_filename : Filename
         Name of output PNG
-    arr : ArrayLike
-        input 2D image array
+    input_filename : Filename
+        Name of input NetCDF file.
+    dataset_name: str
+        Name of datast to be made into a browse image.
     max_dim_allowed : int, default = 2048
         Size (in pixels) of the maximum allowed dimension of output image.
         Image gets rescaled with same aspect ratio.
     """
-    orig_shape = arr.shape
-    scaling_ratio = max([s / max_dim_allowed for s in orig_shape])
-    # scale original shape by scaling ratio
-    scaled_shape = [int(np.ceil(s / scaling_ratio)) for s in orig_shape]
+    # check if dataset can be plotted
+    valid_dataset_names = [
+        "unwrapped_phase",
+        "connected_component_labels",
+        "temporal_coherence",
+        "interferometric_correlation",
+        "persistent_scatterer_mask",
+    ]
```

Comment:
just thinking for the future- perhaps we could move these to `product`, or in some way label/specify in `product.py` which are the raster datasets